packages/medusa/src/services/payment-collection.ts

```diff
@@ -374,7 +392,15 @@ export default class PaymentCollectionService extends TransactionBaseService {
             .catch(() => null)
 
       const paymentSessions: PaymentSession[] = []
-      const inputData: PaymentProviderDataInput = {
+
+      const inputData: PaymentSessionInput = {
+        cart: {
```

Comment:
Is it possible to avoid sending this empty cart object here? We have the email in the customer object.

Reply:
I would like to keep that for the next pr which will be about the API definition around the interfaces and abstract class